README.md

```diff
@@ -175,7 +176,10 @@ const provision = async () => {
     server.views({
         engines: { pug: Pug },
         relativeTo: __dirname,
-        path: 'examples/pug/templates'
+        path: 'examples/pug/templates',
+        compileOptions: {
+            basedir: Path.join(__dirname, 'examples/pug/templates'),
```

Comment:
Could you please add a comment to this `basedir` and the one in `examples/pug/index` as to what this option does?

Reply:
Done, hopefully that makes sense.